packages/cli/cloud/src/utils/helpers.ts

```diff
@@ -0,0 +1,43 @@
+import chalk from 'chalk';
```

Comment:
Can you add a //TODO to remove the duplicated code later with a share CLI pkg or sth

Reply:
ping on this before merge